src/microbe_stage/BiomeConditions.cs

```diff
@@ -58,6 +58,8 @@ public object Clone()
         {
             Compounds = new Dictionary<Compound, EnvironmentalCompoundProperties>(Compounds.Count),
             Chunks = new Dictionary<string, ChunkConfiguration>(Chunks.Count),
+            Sunlight = new DaylightProperties(
+                Sunlight?.Maximum ?? 0.0f, Sunlight?.Current ?? 0.0f, Sunlight?.Average ?? 0.0f),
```

Comment:
I'd like to see this line refactored as this is pretty crazy amount of question marks on a single line. Also this has the effect that all cloned conditions *always* have the sunlight, but in the constructor, this is not created by default, which I assumed means that patches with no sunlight would have the sunlight be null. Why is there a discrepancy?